Content.Server.Database/Content.Server.Database.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\RobustToolbox\MSBuild\Robust.Properties.targets"/>
+  <PropertyGroup>
+    <!-- Work around https://github.com/dotnet/project-system/issues/4314 -->
+    <TargetFramework>$(TargetFramework)</TargetFramework>
+    <LangVersion>8</LangVersion>
+    <IsPackable>false</IsPackable>
+    <Platforms>x64</Platforms>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <OutputPath>..\bin\Content.Server.Database\</OutputPath>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <Nullable>enable</Nullable>
```

Comment:
Look at mr fancy here.

Reply:
Actually I just checked the build log on Travis. There's a bunch of warnings related to nullable reference types (`Model.cs` for one). Please either fix those or disable nullable reference types here.

See https://docs.microsoft.com/en-us/ef/core/miscellaneous/nullable-reference-types maybe.